databricks/koalas/generic.py

```diff
@@ -1279,6 +1279,8 @@ def groupby(self, by, as_index: bool = True):
             col_by = [_resolve_col(df, col_or_s) for col_or_s in by]
             return DataFrameGroupBy(df_or_s, col_by, as_index=as_index)
         if isinstance(df_or_s, Series):
+            if not isinstance(by[0], Series):
```

Comment:
And .. you should also check all instances in `by`, not only the first element.

Reply:
that's right. thanks! :)